examples/pytorch/IterativePruning/README.md

```diff
@@ -0,0 +1,48 @@
+## Iterative Pruning
+Pruning is the process of compressing a neural network that involves removing weights from a trained model.
+It could be achieved by either removing the neurons in a specific layer or make the weights of the connections 
+that are nearer to zero as zero. The script is an example of later. Pruning a model has an impact on
+accuracy of the model, while it makes the model lightweight.
+
+In this example, we train a model to classify MNIST handwritten Digit recognition dataset. Basemodel along with the parameters, metrics and summary are stored in mlflow.
+Subsequently, the trained MNIST model stored in mlflow is saved and pruned iteratively by using the custom 
+inputs provided from the cli. Ax is a platform for optimizing any kind of experiment, including machine learning experiments,
```

Comment:
How and why does Ax come into play? I'm kind of confused - I thought this example was about iterative pruning?

Reply:
Ah, I see we mention that we use Ax for pruning later on. Could we state upfront here that we leverage Ax for iterative pruning to avoid ^such confusion?